src/abis/BlocklistVault.json

```diff
@@ -523,6 +523,44 @@
     "name": "ValidatorsRootUpdated",
     "type": "event"
   },
+  {
+    "anonymous": false,
+    "inputs": [
+      {
+        "indexed": true,
+        "internalType": "address",
+        "name": "caller",
+        "type": "address"
+      },
+      {
```

Comment:
Dont' think we need abis in all the vault types. Should be enough to have in Vault.json only